### DISCUSSION TITLE
Cordova-Android 11,  cordova build android --release

### DISCUSSION THREAD

**Discussion by phusting** [apache/cordova#361] [Q&A]
I was finally able to get everything upgraded to the latest Android platform.  I know that this now builds the bundle file or the app-release.aab for upload and distribution in the Play Store.   My question is, because I won't get to it for a week or two, is when I run the " cordova build android --release" command, it will automatic pull in my keystore information too or do I need to do something different?

**Comment by breautek** [comment]
No, if the key information is not known by Cordova, Cordova will produce unsigned bundles/apks, which will have to be manually signed before uploading to the Google Play store.

See [Signing an App](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#signing-an-app) for more information on how to provide the keystore information to Cordova if you want Cordova to sign the package for you.

Personally I'd recommend creating a [build.json](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#using-buildjson) file, which Cordova automatically looks for and will be used to sign your bundle/APK.
